File: bee_1060.py
# Números Positivos

def main():
    # Entrada
    positivos = 0
    for i in range(6):
        numero = float(input())
        if (numero > 0):
            positivos += 1
    # Saída
    print(f"{positivos} valores positivos")
    
main()

File: bee_1060.c
#include <stdio.h>
 
int main() {
 
    // Entrada
    int positivos = 0;
    for (int i = 0; i < 6; i++) {
        float numero;
        scanf("%f", &numero);
        if (numero > 0) {
            positivos++;
        }
    }
    // Saída
    printf("%d valores positivos\n", positivos);
    
    return 0;
}